src/api/style.js

```diff
@@ -146,6 +146,10 @@ export default class Style {
         return this._styleSpec.order;
     }
 
+    isAnimated() {
+        return this.getColor().isAnimated() || this.getWidth().isAnimated();
```

Comment:
Check strokes too, please

Reply:
I have borrowed this from https://github.com/CartoDB/renderer-prototype/pull/96/files#diff-ba2b399628ae0c09a9b06d41faff9b64L719. I will add the srokes.